src/components/score/score.tsx

```diff
@@ -60,6 +66,38 @@ interface ScoreProps {
 }
 
 export const Score: FC<ScoreProps> = ({ player }) => {
+  const playSfx = usePlaySFX();
+  const loseInt = Math.floor(Math.random() * LOSE_SFX) + 1;
```

Comment:
This approach looks a bit hacky. No need to cast as TrackId.
If you are looking for selecting randomly from the subset of an array, then get the array of TrackIDs, splice it in the known location and get a random element of the subset.

Reply:
Done, used an array